### PR TITLE
Changelog for v2.174.0

### PR DESCRIPTION
**Pull request by vEpiphyte**
- Changelog for v2.174.0
- Update pol:country:currencies to be a uniq + sorted array.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,135 @@
...
 
+v2.174.0 - 2024-07-06
+=====================
+
+Automatic Migrations
+--------------------
+- Renormalize ``ou:position:title``, ``ou:conference:name``, and
+  ``ou:conference:names`` secondary properites.
+  (`#3701 <https://github.com/vertexproject/synapse/pull/3701>`_)
+- Migrate instances of ``econ:currency`` properties into their own nodes.
```
Comment (invisig0th) [inline CHANGELOG.rst:17]:
```suggestion
- Populate new ``econ:currency`` forms from secondary properties.
```

Comment (invisig0th) [inline CHANGELOG.rst:17]:
```suggestion
- Populate new ``econ:currency`` nodes from existing secondary properties.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,135 @@
...
+      The loc of the place where the payment occurred.
+
+    ``place:name``
+      The name of the place where the payment occurred.
+
+  ``pol:country``
+    The form had the following property added to it:
+
+    ``currencies``
+      The name of the place where the payment occurred.
```
Comment (invisig0th) [inline CHANGELOG.rst:80]: If this was auto-populated there may be a copy-pasta error. This should be more like "The currencies officially used by the country."

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,135 @@
...
+
+  **Updated Properties**
+
+  ``ou:position``
+    The form had the following property updated on it:
+
+    ``title``
+      This property is now a ``entity:name`` type.
```
Comment (invisig0th) [inline CHANGELOG.rst:88]:
```suggestion
      This property is now an ``entity:name`` type.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,135 @@
...
+
+  ``ou:conference``
+    The form had the following properties updated on it:
+
+    ``name``
+      This property is now a ``entity:name`` type.
```
Comment (invisig0th) [inline CHANGELOG.rst:94]:
```suggestion
      This property is now an ``entity:name`` type.
```

File: synapse/models/geopol.py [hunk 5]
```diff
@@ -89,7 +89,7 @@ def getModelDefs(self):
                         ('vitals', ('pol:vitals', {}), {
                             'doc': 'The most recent known vitals for the country.'}),
 
-                        ('currencies', ('array', {'type': 'econ:currency'}), {
+                        ('currencies', ('array', {'type': 'econ:currency', 'sorted': True, 'uniq': True}), {
```
Comment (invisig0th) [inline synapse/models/geopol.py:92]: Thanks!

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (vEpiphyte) [reply CHANGELOG.rst]: copy-pasta error :spaghetti:

**Review by invisig0th — APPROVED** [review]

File: CHANGELOG.rst [hunk 6]
```diff
@@ -6,6 +6,135 @@
 Synapse Changelog
 *****************
 
+v2.174.0 - 2024-07-08
+=====================
+
+Automatic Migrations
+--------------------
+- Renormalize ``ou:position:title``, ``ou:conference:name``, and
+  ``ou:conference:names`` secondary properites.
```
Comment (Cisphyx) [inline CHANGELOG.rst:15]:
```suggestion
  ``ou:conference:names`` secondary properties.
```

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**